bluebottle/projects/migrations/0051_auto_20171113_1637.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2017-11-13 15:37
+from __future__ import unicode_literals
+
+import bluebottle.utils.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0050_merge_20171110_1633'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='projectplatformsettings',
+            options={'verbose_name': 'project platform settings', 'verbose_name_plural': 'project platform settings'},
+        ),
+        migrations.AddField(
+            model_name='projectplatformsettings',
+            name='allow_anonymous_rewards',
+            field=models.BooleanField(default=True),
+        ),
+        migrations.AlterField(
+            model_name='project',
+            name='theme',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to='bb_projects.ProjectTheme'),
+        ),
+        migrations.AlterField(
+            model_name='projectdocument',
+            name='file',
+            field=bluebottle.utils.fields.PrivateFileField(max_length=110, upload_to=b'private/private/private/projects/documents'),
+        ),
```

Comment:
This is wrong of course. It should be fixed in https://github.com/onepercentclub/bluebottle/pull/3002

Reply:
Maybe just remove the AlterField migrations and only keep the AddField?